lib/Twig/SimpleFilter.php

```diff
@@ -11,6 +11,8 @@
 
 /**
  * For Twig 1.x compatibility.
+ *
+ * @deprecated to be removed in 3.0
```

Comment:
this is the wrong place for it. It should be added on the class definition below.